dev-cpp/gtest/gtest-9999.ebuild

```diff
@@ -36,6 +36,13 @@ pkg_setup() {
 	use test && python-any-r1_pkg_setup
 }
 
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i -e 's:-Wall -Wshadow -Werror:-Wall -Wshadow:' \
```

Comment:
Why do you need to repeat the two other flags?

Reply:
It's there to provide context.  With something like `sed -i -e  's:-Werror::'`, I assume the possibility it could edit an instance of `-Werror` in a comment, which would not be what I intend.  And if at a later date upstream decides to selectively add something benign like `-Werror=implicit-fallthrough`, blindly deleting `-Werror` would cause an unnecessary build error.  So, I tend to treat sed as a footgun that should be used with some degree of precision.

If my reasoning is wrong, let me know and I'll change it.

Reply:
Your reasoning is correct, just implementation is poor. The `-Wall -Wshadow` context is of little value, and it will still fail if upstream changes that to `-Wall -Wshadow -Werror=something`. So go for just `s:-Werror::`, and maybe add some context via `/foo/` before sed command, like command/variable name.

Reply:
OK.